plugins/chan_track.py

```diff
@@ -727,7 +727,8 @@ def _parse_mode_string(modes, params, status_modes, mode_types):
                 mode_type = 'B' if is_status else None
 
             if mode_type in "AB" or (mode_type == 'C' and adding):
-                param = params.pop(0)
+                if len(params) != 0:
```

Comment:
```suggestion
                if not params:
```
Comparing the length of an iterable is unnecessary, a bool check is cleaner